code/Data exploration.R

```diff
@@ -78,5 +117,49 @@ data <- data %>%mutate(WEEKDAYWEEKEND = case_when(
   DaysOfWeek %in%  6:7  ~ "WEEKEND"
 ))
 
+##Create DATE and MONTHDATE field for joining/plotting
+data$DATE <- as.Date(substr(data$DATETIME,1,10))
+data$YEARMONTH<- as.Date(paste(substr(data$DATE,1,7), "-01", sep=""))
```

Comment:
this is pretty jank, instead consider `year()` and `month()` functions from `lubridate` package to extract month and year

Reply:
made the change in the latest revision. 👍